pkg/policy/policy_controller.go

```diff
@@ -389,6 +399,340 @@ func (pc *PolicyController) getPolicy(key string) (kyvernov1.PolicyInterface, er
 	}
 }
 
+func (pc *PolicyController) processExistingResources(policy kyvernov1.PolicyInterface) {
```

Comment:
Do we still need this? Looks like it only reports events.

Reply:
I think yes, will check in another PR.